Fix missing build-time error for asUrl() (#1068)
inrupt/solid-client-js: CHANGELOG.md

## [Unreleased]

### Bugs fixed

- Since version 1.8.0, TypeScript would no longer warn you if you omitted the

>   second argument to `asUrl` in cases where it was should not be omitted,

Reviewer: Just to void the double negative:
```suggestion
  second argument to `asUrl` in cases where it's required,
```

Author: 2ba8d5ee
